### PR TITLE
Adds the lizard-integrated lighter

### PR DESCRIPTION
**Pull request by super12pl**
## About The Pull Request
~~part of the lizard rework~~

So I wanted to add some gimmicky stuff to lizards, as they don't really have much mechanics-wise and thats what I came up with.

Lizards can now drink welding fuel(at least 4u) then use an action button to be able to "spit fire" in a lighter-manner.

If anyone wondering balance, you can already get lighters that don't require drinking industrial fuel for free in gear shop.

## Why It's Good For The Game
Flavour/it's pretty cool
Also lizards having dormant firebreath gene makes more sense now I suppose.

## Changelog
:cl:
add: Lizards can now "spit fire"(in a lighter manner) by drinking welding fuel.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species_types/lizardpeople.dm [hunk 1]
```diff
@@ -34,6 +35,42 @@
...
+	else
+		qdel(N)
+		to_chat(H, "<span class='warning'>You don't have any free hands.</span>")
+
+/datum/action/innate/lizlighter/IsAvailable()
+	if(..())
+		var/mob/living/carbon/human/H = owner
+		if(H.reagents && H.reagents.has_reagent(/datum/reagent/fuel,4))
+			return 1
+		return 0
```
Comment (ZephyrTFA) [inline code/modules/mob/living/carbon/human/species_types/lizardpeople.dm]: TRUE / FALSE

Comment (super12pl) [reply code/modules/mob/living/carbon/human/species_types/lizardpeople.dm]: done and done

**Review by MarkSuckerberg — APPROVED** [review]

File: code/modules/mob/living/carbon/human/species_types/lizardpeople.dm [hunk 2]
```diff
@@ -22,6 +22,7 @@
 	disliked_food = GRAIN | DAIRY
 	liked_food = GROSS | MEAT
 	inert_mutation = FIREBREATH
+	var/datum/action/innate/lizlighter/lizlighter
```
Comment (ZephyrTFA) [inline code/modules/mob/living/carbon/human/species_types/lizardpeople.dm]: This is mostly nitpick, but can you move the new var declaration below existing vars being set

Comment (super12pl) [reply code/modules/mob/living/carbon/human/species_types/lizardpeople.dm]: sure

**Pull request merged**